### DISCUSSION TITLE
require for quantification algorithm

### DISCUSSION THREAD

**Discussion by RichYang111** [vdemichev/DiaNN#967] [General]
Dear supporter, 
Recently, i read a paper which analyzed the DIA raw data using DIA NN software, but i saw the researcher used peak height algorithm to quantification the peptides.  I looked for some information about the algorithm introduction on the GIT HUB interface, but there is no any information about height peak algorithm. Would you please give me some  explanation.  Thanks  alot

**Comment by vdemichev** [comment]
The algorithm is quite simple, it sums the XIC values at the apex and at the adjacent scans.
